dist/src/main/assembly/conf/brooklyn/default.catalog.bom

```diff
@@ -30,11 +30,11 @@ brooklyn.catalog:
         docker.policy.ha.enable: false
         docker.container.strategies:
         - $brooklyn:object:
-            type: clocker.docker.location.strategy.MaxContainersPlacementStrategy
+            type: clocker.docker.location.strategy.basic.MaxContainersPlacementStrategy
```

Comment:
This kind of change breaks backwards compatibility of persisted state - we need to be increasingly careful of that. One solution is to use the Brooklyn `DeserializingClassRenamesProvider`. We can add to the `classpath://org/apache/brooklyn/core/mgmt/persist/deserializingClassRenames.properties`.

Perhaps we want to improve Brooklyn so that it's not just a single file, but instead we also pick up all properties files in some well-defined classpath folder.